### PR TITLE
Added baseline docs for schedule config

### LINKED ISSUE #424: Document Scheduler integration
As a developer, I'd like to document the integration of PCF Scheduler + SCDF, so it'd help the users to follow along, configure, and set it up for use.

Depends on screenshots via spring-cloud/spring-cloud-dataflow#297.

**Acceptance:**
- Document configuration properties exposed via SCSh-CF-Scheduler and/or CF-Deployer
- Include a sample manifest showing the new properties 
- Include a new section that delves into the specifics of architecture with a simple block diagram
- Link to Dashboard screenshots from the core docs via spring-cloud/spring-cloud-dataflow#297

### PR DESCRIPTION
**Pull request by cppwfs**
resolves #424

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc [hunk 1]
```diff
@@ -614,3 +614,43 @@ services:
 - mysql
 - my-config-server #this is for the server
 ----
+
+[[getting-started-scheduling-configuration]]
+== Configure Scheduling
+In this section we will discuss how to configure Spring Cloud Data Flow to connect to the https://www.cloudfoundry.org/the-foundry/scheduler/[PCF-Scheduler] as its agent to execute tasks.
+
+NOTE: So before following the instructions below, be sure to have an instance of the PCF-Scheduler service running in your Cloud Foundry space.  To create a PCF-Scheduler in your space (assuming it is in your Market Place) execute the following from the cf CLI: `cf create-service scheduler-for-pcf standard <name of service>`
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc]: We can remove `So` ;)

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc [hunk 2]
```diff
@@ -614,3 +614,43 @@ services:
...
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_URL: https://api.local.pcfdev.io
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_ORG: pcfdev-org
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SPACE: pcfdev-space
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_DOMAIN: local.pcfdev.io
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_USERNAME: admin
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_PASSWORD: admin
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_STREAM_SERVICES: rabbit
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_TASK_SERVICES: mysql,myscheduler
+    SPRING_CLOUD_DEPLOYER_CLOUDFOUNDRY_SKIP_SSL_VALIDATION: true
+    SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES-ENABLED: true
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc]: @cppwfs I believe this property needs to be `SPRING_CLOUD_DATAFLOW_FEATURES_SCHEDULES_ENABLED` - similar to how we set the other feature flags. Could you confirm?

Comment (cppwfs) [reply spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/configuration.adoc]: Great Catch, Thanks!

**Comment by ilayaperumalg** [comment]
LGTM, merging.

**Comment by ilayaperumalg** [comment]
Squashed and merged as 2c2fb84

**Pull request closed without merging**